server/server.js

```diff
@@ -11,7 +11,12 @@ const PORT = process.env.PORT || 3001;
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  context: authMiddleware,
+  context: ({req}) => {
+    const authorization = req.headers.authorization || '';
+    return {
+      authorization
+    };
+  }
```

Comment:
The authMiddleware method was requiring every mutation and query to be logged in with a token. Mutations and Queries such as login and signup do not require a token. Hence the issue you were encountering.